packages/dappmanager/src/modules/chains/drivers/monero.ts

```diff
@@ -1,5 +1,5 @@
 // @ts-ignore
-import Daemon from "monero-rpc";
+import { Daemon } from "monero-rpc";
```

Comment:
Looks like now you can delete the `// @ts-ignore`

Reply:
bump @dsimog01

Reply:
ts-ignore is still needed. The build fails and there is no `@types/monero-rpc`